### DISCUSSION TITLE
Skin editor in multi spectator doesnt work as intended

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#24748] [Q&A]
If you place components with no specific player zoomed in it works correctly, but with gameplay screen selected it doesn't work at all. Editor places components at a random place.

https://github.com/ppy/osu/assets/86934170/2c8982c8-4e42-43df-a43d-aca5d5d6659b

**Comment by peppy** [comment]
Aware. Don't use it for now.
